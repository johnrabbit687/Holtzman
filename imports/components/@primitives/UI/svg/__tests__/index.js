import renderer from "react-test-renderer";
import Svg from "../index";

describe("Svg Component", () => {
  it("should exist", () => {
    const svg = renderer.create(
      <Svg name={"home"} />
    );
    expect(svg).toBeDefined();
  });

  it ("should render a home svg", () => {
    const homeSvg = renderer.create(
      <Svg name={"home"} />
    );
    expect(homeSvg).toMatchSnapshot();
  })
})