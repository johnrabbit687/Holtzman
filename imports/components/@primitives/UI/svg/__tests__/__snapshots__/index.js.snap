exports[`Svg Component should render a home svg 1`] = `
<svg
  height="24px"
  viewBox="0 0 24 24"
  width="24px"
  xmlns="http://www.w3.org/2000/svg"
  xmlnsXlink="http://www.w3.org/1999/xlink">
  <title />
  <defs>
    <path
      d="M10 20v-6h4v6h5v-8h3L12 3 2 12h3v8z"
      id="a" />
  </defs>
  <g
    fill="none"
    fillRule="evenodd">
    <path
      d="M0 0h24v24H0z" />
    <mask
      fill="#fff"
      id="b">
      <use
        xlinkHref="#a" />
    </mask>
    <use
      fill="#444"
      fillRule="nonzero"
      xlinkHref="#a" />
    <path
      d="M0 0h24v24H0z"
      fill="#1C683E"
      mask="url(#b)" />
  </g>
</svg>
`;
